ci-operator/step-registry/quay-tests/test/performance/quay-tests-test-performance-commands.sh

```diff
@@ -0,0 +1,237 @@
+          - name: ES_PORT
+            value: "443"
+          - name: PYTHONUNBUFFERED
+            value: "0"
+          - name: ES_INDEX
+            value: "quay-vegeta"
+          - name: PUSH_PULL_IMAGE
+            value: "quay.io/quay-qetest/quay-load:latest"
+          - name: PUSH_PULL_ES_INDEX
+            value: "quay-push-pull"
+          - name: PUSH_PULL_NUMBERS
+            value:  "${PUSH_PULL_NUMBERS}"
+          - name: TARGET_HIT_SIZE
+            value: "${HITSIZE}"
+          - name: CONCURRENCY
+            value: "${CONCURRENCY}"
+          - name: TEST_NAMESPACE
+            value: "${quay_perf_namespace}"
+          - name: TEST_PHASES
+            value: "${TEST_PHASES}"
+            # value: "LOAD,RUN,DELETE"
+        resources:
+          requests:
+            cpu: "1"
+            memory: "512Mi"
+        imagePullPolicy: Always
+      restartPolicy: Never
+  backoffLimit: 0
+
+EOF
+
+echo "the Perf Job needs about 3~4 hours to complete"
+echo "check the OCP Quay Perf Job, if it complete, go to Kibana to generate index pattern and get Quay Perf metrics"
+
+#wait until the quay perf testing job complete, and show the job status
+oc get job -n "$quay_perf_namespace"
+oc -n "$quay_perf_namespace" wait job/quay-perf-test-orchestrator --for=jsonpath='{.status.ready}'=0 --timeout=600s
+
+# 3, Wait until the job complete
+
+date
+start_time=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
+
+quayperf_pod_name=$(oc get pod -l job-name=quay-perf-test-orchestrator -n ${quay_perf_namespace} -o jsonpath='{.items[0].metadata.name}')
+echo "$quayperf_pod_name"
+
+if [[ -z "${quayperf_pod_name}" ]]; then
+  echo "No quay-perf-test-orchestrator pod started, please check"
+  exit 1
+fi
+
+sleep 120 #wait pod start
+
+TEST_UUID=$(oc logs "$quayperf_pod_name" -n "${quay_perf_namespace}" | grep 'test_uuid' | sed -n 's/^.*test_uuid=\s*\(\S*\).*$/\1/p')
+echo "$TEST_UUID"
+echo "job start: $start_time"
+
+JOB_STATUS="Success"
+oc wait --for=condition=complete --timeout=6h job/quay-perf-test-orchestrator -n "$quay_perf_namespace"
+if [ $? -ne 0 ]; then
+  JOB_STATUS="Failed"
+fi
+date
+
+end_time=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
+
+echo "job end $end_time and status $JOB_STATUS"
+
+# fetch UUID,JOB_START etc required data to dashboard http://dashboard.apps.sailplane.perf.lab.eng.rdu2.redhat.com/
+echo "The Prow Job ID is: $PROW_JOB_ID"
+# echo "The Prow Job URL is: $PROW_JOB_URL"
+
+
+# 4, Send the performance test data to ELK
+# https://github.com/cloud-bulldozer/e2e-benchmarking/blob/master/utils/index.sh
+
+# ES_INDEX=perf_scale_ci
+
+# invoke send to dashboad index.sh
+export ES_SERVER="${ELK_SERVER}"
+export BUILD_ID="${BUILD_ID}"
+export UUID="${TEST_UUID}"
+# export BUILD_URL="${PROW_JOB_URL}"
+export JOB_STATUS="$JOB_STATUS"
+export JOB_START="$start_time"
+export JOB_END="$end_time"
+export WORKLOAD="quay-load-test"
+export RELEASE_STREAM="${QUAY_OPERATOR_CHANNEL}"
+export TEST_PHASES="${TEST_PHASES}"
+export hitsize="${HITSIZE}"
```

Comment:
can we just remove these 3 lines and have these vars directly exported please? That should work with the script change here: https://github.com/quay/quay-tests/pull/273
```suggestion
export HITSIZE
export CONCURRENCY
export PUSH_PULL_NUMBERS
```

Reply:
are you sure? why export the variables with empty value?